Solutions/VirusTotal/Playbooks/Get-VirusTotalURLReport/url-alert-trigger/azuredeploy.json

```diff
@@ -360,8 +386,8 @@
                 "[resourceId('Microsoft.Logic/workflows', parameters('PlaybookName'))]"
             ],
             "properties": {
-                "roleDefinitionId": "[concat('/subscriptions/', variables('ASSubscriptionId'), '/', variables('ASResourceGroup'), '/providers/Microsoft.Authorization/roleDefinitions/ab8e14d6-4a74-4a29-9ba8-549422addade')]",
-                "principalId": "[reference(concat(resourceId('Microsoft.Logic/workflows', parameters('PlaybookName')), '/providers/Microsoft.ManagedIdentity/Identities/default'), '2018-11-30').principalId]"
+                "roleDefinitionId": "[[variables('_roleDefinitionId')]",
```

Comment:
why are we adding an extra [ bracket in source file? please revert this change

Reply:
removed extra bracket